index.html

```diff
@@ -64,34 +63,35 @@ <h4>Main Program</h4>
       <div class="h4bottom"></div> 
      </div> 
       <div class="grid2">
-      <div class="prog1">
+      <article class="prog1">
         <img alt="lecture" src="./Features/images/program_icon_01 1.svg" class="imagexxl">
         <p class="word">Lecture</p>
```

Comment:
- [ ] like the previous reviewer noted, kindly align the logo with the content of the page on the left, like in the original design. 
- [ ] please make the main title text weight bolder and add the font size if possible, so it looks like the original design. Also, kindly reduce the width of the text in the rectangle under the main title as proposed by the previous reviewer.
yours
![Screenshot (1176)_LI](https://github.com/Bupilipili/First-Capstone-Project/assets/65298048/bce8b3ca-4133-4196-9072-1701db65c174)
original
![Screenshot (1177)](https://github.com/Bupilipili/First-Capstone-Project/assets/65298048/e5cfb566-c09e-45e8-b6c3-c18583006295)